kotlin/reakt-native-toolkit-ksp/src/main/kotlin/de/voize/reaktnativetoolkit/ksp/processor/ToolkitSymbolProcessor.kt

```diff
@@ -239,6 +246,51 @@ class ToolkitSymbolProcessor(
             }
         }
 
+        val rnViewManagers = resolver.getSymbolsWithAnnotation("$toolkitPackageName.annotation.ReactNativeViewManager")
+            .map { annotatedNode ->
+                when (annotatedNode) {
+                    is KSFunctionDeclaration -> annotatedNode.also {
+                        // this should not be outside RN view manager processing so that projects
+                        // not using compose do not fail on this because Composable is not found
+                        val composableAnnotationType = resolver.getClassDeclarationByName("androidx.compose.runtime.Composable")
```

Comment:
Can we do this with lazy, so only if there is an annotated component we load this

Reply:
done